management-server/go.mod

```diff
@@ -0,0 +1,24 @@
+module github.com/wso2/apk/APKManagementServer
```

Comment:
shall we change the module name as `github.com/wso2/apk/management-server` like the same directory name.